Cloud_Region_Metadata_specification.md

```diff
@@ -0,0 +1,110 @@
+| cfe-region                                           | string                     | “South Korea”         | Carbon Free Energy grid region name as reported by the cloud provider.                                                                                                                                            |
+| em-zone-id                                           | string                     | “KR”                  | Electricity Maps zone identifier for this region. Can be used to get real time data from their API.                                                                                                               |
+| wt-region-id                                         | string                     | “KOR”                 | WattTime region identifier. Can be used to get real time data from their API.                                                                                                                                     |
+| location                                             | string                     | “Seoul”               | Location of the region, as reported by the cloud provider.                                                                                                                                                        |
+| geolocation                                          | numeric, numeric            | 37.532600, 127.024612 | Latitude and longitude of the location, city level, not exact datacenter coordinates.                                                                                                                             |
+| provider-cfe-hourly                                  | numeric proportion 0.0-1.0 | 0.31                  | Carbon Free Energy proportion for this cloud provider and region, weighted by the hourly usage through the year.                                                                                                  |
+| provider-cfe-annual                                  | numeric proportion 0.0-1.0 | 0.28                  | Carbon Free Energy proportion for this cloud provider and region, calculated on an annual totals basis.                                                                                                           |
+| power-usage-effectiveness                            | numeric                    | 1.18                  | Power Usage Effectiveness (PUE) ratio for the region, averaged across individual datacenters.                                                                                                                     |
+| water-usage-effectiveness                            | litres/kWh                 | 2.07                  | Water Usage Effectiveness in litres per kilowatt hour for the region, averaged across individual datacenters.                                                                                                     |
```

Comment:
When we talk about water usage, there are three possible numbers that might be commonly reported here:

* **withdrawn water** - i.e. water taken from a river / aquifer etc 
* **discharged water** - water returned, that might be reused, but might be warmer / cooler than before
* **consumed water** - water withdrawn that was rendered to state that can't be re-used (ie. water that is evapourated, and so on)

Consumed water can't be used by anyone else, so in the context of water being taken from an existing stock that might otherwise be used for drinking water of argiculture the distinction is meaningful.

Also, because consumed water is basically withdrawn water minus discharged water, it will usually be lower than the other two measures.

As I understand it, the Microsoft Azure figure is for _withdrawn_ water. Can we make this explicit in the table to avoid cases where we are comparing a large "withdrawn" number from one provider, to a small "consumption" number from another?